Sources/ShapedArray/ShapedArray+ExpressibleByArrayLiteral.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ShapedArray+ExpressibleByArrayLiteral.swift
```

Comment:
We should use "+" in filenames sparingly. The Swift Stdlib never uses it, although `swift-numerics` and `swift-collections` do. In fact, `DequeModule` has [a file with this naming convention](https://github.com/apple/swift-collections/blob/main/Sources/DequeModule/Deque%2BExpressibleByArrayLiteral.swift).